app/interview/select/[devType]/prepare/page.tsx

```diff
@@ -2,14 +2,17 @@ import TopicSelector from '@/app/_components/TopicSelector';
 import { DeveloperType } from '@/app/_types/interview';
 import { isDeveloperType } from '@/app/_utils/typeGuards';
 import { DEVELOPER_OPTIONS } from '@/app/interview/_constants/developers';
-import SelectLayout from '@/app/interview/select/layout';
+import SelectLayout from '@/app/interview/select/_components/SelectLayout';
 
 interface PreparePageType {
-  params: { devType: string };
-  searchParams: { topics?: string };
+  params: Promise<{ devType: string }>;
+  searchParams: Promise<{ topics?: string }>;
 }
 
-export default function PreparePage({ params: { devType }, searchParams: { topics } }: PreparePageType) {
+export default async function PreparePage({ params, searchParams }: PreparePageType) {
+  const { devType } = await params;
+  const { topics } = await searchParams;
```

Comment:
```suggestion
const [{ devType }, { topics }] = await Promise.all([params, searchParams]);
```

위 방법이 되는지 한번 확인 부탁드립니다~

Reply:
PR 제목도 한번 확인 부탁드립니당~

Reply:
대박!Promise all도 되네요! 좋다!